pulseapi/settings.py

```diff
@@ -32,20 +32,25 @@
     CORS_ORIGIN_REGEX_WHITELIST=(list, []),
     CORS_ORIGIN_WHITELIST=(list, []),
     CSRF_TRUSTED_ORIGINS=(list, []),
+    DATABASE_URL=(str, None),
```

Comment:
sorted alphabetically, only the `USE_RECAPTCHA` and `RECAPTCHA_SECRET` values are new.